pkg/crc/network/template.go

```diff
@@ -7,13 +7,13 @@ import (
 
 const (
 	resolvFileTemplate = `# Generated by CRC
-search {{ range .SearchDomains }}{{ .Domain }}{{ end }}
+{{ range .SearchDomains }}search {{ .Domain }}{{ end }}
 {{ range .NameServers }}nameserver {{ .IPAddress }}
```

Comment:
Why is this changed?

Reply:
@cfergeau this is changed because if there is no search in the `/etc/resolv.conf` then end result after processing the template will be `search ` without anything and that will cause issue for command like dig to parse it.

Reply:
Ah I see. This could almost go in a separate commit as this is an unrelated bugfix.. Is this also behaving as expected when multiple search domains are in use?

Reply:
yes now it behave as expected even multiple search domains are in use.